docs/Passage/Authentication.md

```diff
@@ -0,0 +1,58 @@
+# Passage\Controllers\Authentication
+
+All URIs are relative to https://api.passage.id/v1, except if the operation defines another base path.
+
+## `validateJWT()`
+
+```php
+validateJWT($jwtString): string | null
+```
+
+Validate JWT string
+
+Valid JWT string returns user id.
+
+### Laravel Example
+
+```php
+<?php
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+use PassageIdentity\PassagePHP\Passage\Controllers\Authentication;
+
+// Authentication using Passage class instance
+Route::get('authenticatedRoute', function (Request $request) {
+    $passage = new Passage(env('APP_ID'), env('API_KEY'));
+    $jwtString = $request->header('psg_auth_token');
```

Comment:
In the old SDKs we had some helpers to grab it from headers by default but then a param you could pass to say cookies instead. 

Do we need to put that more clearly in the docs now that we aren't doing the request inspection side of this flow?

Reply:
I don't think we should compare the SDKs. It would be surprising if they are comparing SDKs. If they are here in these docs then they are using PHP and this is the only option. 

With that being said, I do think the difference should be noted in the Gitbook docs on the tokens page.